### ISSUE TITLE
A configuration parameter missing in the examples.

### ISSUE THREAD

**Issue by magnus-eriksson** [mrjgreen/database#3]
I just noticed that the configuration examples in the doc for MySQL are missing the `database`-parameter for creating a connection. :)

**Comment by mrjgreen** [comment]
That shouldn't be a problem - by design the component works across schemas. 

Cheers
Joe Green

**Comment by dotnetCarpenter** [comment]
Well, it doesn't work for the MySqlConnector.

I was struggling with this issue for little while and didn't get it until I stepped through the code with my debugger. I think it would help a lot to mention it.

**Comment by magnus-eriksson** [comment]
You can add the schema in the `table()` call: `$connection->table('databaseName.tableName')` or like you usually do in the query: `$connection->query('SELECT * FROM databaseName.tableName')`. But I agree that this is not well documented, though.

**Comment by mrjgreen** [comment]
Makes sense, I'll update the docs. However, I do think its good practice to fully qualify table names to avoid ambiguity, and connecting to a "default" schema can sometimes cause confusion.

Its been a while since I've only been dealing with a single schema though, and I understand thats not necessarily the case for everyone.
